frontend/src/components/Footer.vue

```diff
@@ -0,0 +1,99 @@
+<script setup lang="ts"></script>
+
+<template>
+    <div id="footer">
+        <div class="container">
+            <div class="icons">
+                <a class="icon" href="https://fr-fr.facebook.com/BeninExcellence/">
+                    <i class="fa fa-facebook-f" />
+                </a>
+                <a class="icon" href="https://twitter.com/excellencebenin">
+                    <i class="fa fa-twitter" />
+                </a>
+                <a class="icon" href="#i">
+                    <i class="fa fa-instagram" />
+                </a>
+                <a class="icon" target="_blank" rel="noreferrer"
+                    href="https://www.linkedin.com/company/benin-excellence/">
+                    <i class="fa fa-linkedin" />
+                </a>
+            </div>
+            <ul class="copyright">
+                <li>&copy; Fondation Vallet</li>
+                <li>
+                    <a target="_blank" rel="noreferrer" href="https://www.fondationvallet.org/eeia/">
+                        École d'Été en Intelligence Artificielle (EEIA)
+                    </a>
+                </li>
+                <li>
+                    <a target="_blank" rel="noreferrer" href="#i">
+                        Bénin Excellence
+                    </a>
+                </li>
+            </ul>
+        </div>
+    </div>
+</template>
+
+<style scoped>
+#footer {
+    background-color: #f6f6f6;
+}
+
+.icons {
+    color: #aaa;
+    font-size: 2.7em;
+    text-align: center;
+    padding-top: 1em;
+    padding-bottom: .5em;
+}
+
+.icon {
+    color: #aaa;
+    margin-left: 15px;
+    margin-right: 15px;
+    cursor: pointer;
+}
+
+.icon:hover,
+.icon:active {
+    color: #555;
+}
+
+.copyright {
+    list-style: disc;
+    color: #aaa;
+    text-align: center;
+    margin: 0;
+    padding: 0;
+    padding-bottom: 15px;
+}
+
+.copyright li:first-child {
+    border: none;
+}
+
+.copyright li {
+    display: inline-block;
+    border-left: solid 1px rgba(144, 144, 144, 0.25);
+    padding: 15px;
+}
+
+@media only screen and (max-width: 600px) {
```

Comment:
This is how you do mobile version. 

<img width="477" alt="Screenshot 2022-11-08 at 14 25 18" src="https://user-images.githubusercontent.com/58826859/200576417-0d5939c6-d7b3-4a0a-af5f-192e1fd00e08.png">